app/Resources/views/page.html.twig

```diff
@@ -220,7 +220,7 @@
         </noscript>
     {% endif %}
 
-    <div class="global-wrapper{% if isMagazine ?? false %} page--magazine{% endif %}{% if clean ?? false %} clean{% endif %}" data-behaviour="FragmentHandler Math{% if hypothesis ?? false %} HypothesisLoader{% endif %}"
+    <div class="global-wrapper{% if isMagazine ?? false %} page--magazine{% endif %}{% if hasTwoColumns ?? false %} social-media-page--wrapper{% endif %}{% if clean ?? false %} clean{% endif %}" data-behaviour="FragmentHandler Math{% if hypothesis ?? false %} HypothesisLoader{% endif %}"
```

Comment:
it's a little confusing that the class that is introduced when `hasTwoColumns` is social-media-page--wrapper. Not blocking, just feedback.

Reply:
The condition is changed from hasTwoColumns to hasSocialMedia.